environment/src/main/resources/application.yml

```diff
@@ -145,7 +145,9 @@ cb:
         services: s3,dynamodb
       interface:
         services: autoscaling,cloudformation,ec2,ecr.api,ecr.dkr,elasticfilesystem,elasticloadbalancing,sts
-    credential.cache.ttl: 60
+    credential:
+      cache.ttl: 60
+      default.regions: us-west-1,us-west-2,eu-central-1,eu-north-1,sa-east-1,ap-northeast-1,eu-west-1,ap-northeast-2,ap-southeast-2,ap-southeast-1
```

Comment:
hmmm
I am not sure this is the approach what we would like to follow. How about using aws-zone.json for this ?

Reply:
hm, that is not configurable without a new version